tests/conftest.py

```
@@ -148,3 +88,91 @@ def ezkl_available():
+def pre_sliced_doom(models_root, session_cache_dir):
+    out = session_cache_dir / "doom_sliced"
+    return _do_slice(models_root / "doom", out)
+
+
+@pytest.fixture(scope="session")
+def pre_sliced_doom_tiled(models_root, session_cache_dir):
+    out = session_cache_dir / "doom_tiled_sliced"
+    return _do_slice(models_root / "doom", out, tile_size=1000)
+
+
+def _do_compile(src, dest, session_cache_dir, **kwargs):
```

Ruff (0.14.14)

[warning] Unused function argument: `session_cache_dir`

(ARG001)